sdk/helper/strutil/strutil.go

```diff
@@ -32,6 +32,16 @@ func StrListContains(haystack []string, needle string) bool {
 	return false
 }
 
+// StrListContainsCaseInsensitive looks for a string in a list of strings.
+func StrListContainsCaseInsensitive(haystack []string, needle string) bool {
```

Comment:
love the variable names :)

Reply:
Yeah, that's pretty common in other APIs.  Suprised Go didn't use it for things like strings.Contains